### PR TITLE
dphys-swapfile: add page

### PR DESCRIPTION
**Pull request by MDecker-MobileComputing**
- [X] The page (if new), does not already exist in the repository.
- [X] The page is in the correct platform directory (`common/`, `linux/`, etc.)
- [X] The page has 8 or fewer examples.
- [X] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- [X] The page follows the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [X] The page description includes a link to documentation or a homepage (if applicable).

### CODE REVIEW

File: pages/linux/dphys-swapfile.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+
+- Disable swap file:
+
+`dphys-swapfile swapoff`
+
+- Enable swap file:
+
+`dphys-swapfile swapon`
+
+- Create new swap file (e.g., after changes in file `/etc/dphys-swapfile `):
```
Comment (CleanMachine1) [inline pages/linux/dphys-swapfile.md:14]:
```suggestion
- Create a new swap file:
```

**Review by CleanMachine1 — CHANGES_REQUESTED** [review]

File: pages/linux/dphys-swapfile.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
+# dphys-swapfile
+
+> Manage the swap file on Debian-based Linux systems.
+> More information: <https://manpages.debian.org/jessie/dphys-swapfile/dphys-swapfile.8.en.html>.
```
Comment (navarroaxel) [inline pages/linux/dphys-swapfile.md:4]: This link will be on sync with the latest version
```suggestion
> More information: <https://manpages.debian.org/latest/dphys-swapfile/dphys-swapfile.html>.
```

**Review by marchersimon — APPROVED** [review]
Thanks @MDecker-MobileComputing

**Review by navarroaxel — APPROVED** [review]

**Review by sbrl — APPROVED** [review]
Thanks, @MDecker-MobileComputing!

**Comment by sbrl** [comment]
Merging anyway, as @CleanMachine1's comments appear to have been addressed.

**Pull request merged**